src/Functions.Worker.Extensions.OpenAI/Embeddings/EmbeddingsStoreInputAttribute.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.OpenAI.Embeddings;
+
+public class EmbeddingsStoreInputAttribute : InputBindingAttribute
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EmbeddingsStoreAttribute"/> class with the specified connection
+    /// and collection names.
+    /// </summary>
+    /// <param name="input">The input source containing the data to generate embeddings for.</param>
+    /// <param name="inputType">The type of the input.</param>
+    /// <param name="title">The title of the document to be stored.</param>
+    /// <param name="connectionName">
+    /// The name of an app setting or environment variable which contains a connection string value.
+    /// </param>
+    /// <param name="collection">The name of the collection or table to search or store.</param>
+    /// <exception cref="ArgumentNullException">
+    /// Thrown if <paramref name="input"/> or <paramref name="title"/> or <paramref name="collection"/> or <paramref name="connectionName"/> are null.
+    /// </exception>
+    public EmbeddingsStoreInputAttribute(string input, InputType inputType, string title, string connectionName, string collection)
+    {
+        this.Input = input ?? throw new ArgumentNullException(nameof(input));
+        this.InputType = inputType;
+        this.Title = title ?? throw new ArgumentNullException(nameof(title));
+        this.ConnectionName = connectionName ?? throw new ArgumentNullException(nameof(connectionName));
+        this.Collection = collection ?? throw new ArgumentNullException(nameof(collection));
+    }
+
+    /// <summary>
+    /// Gets or sets the ID of the model to use.
+    /// </summary>
+    /// <remarks>
+    /// Changing the default embeddings model is a breaking change, since any changes will be stored in a vector database for lookup. Changing the default model can cause the lookups to start misbehaving if they don't match the data that was previously ingested into the vector database.
+    /// </remarks>
+    public string Model { get; set; } = OpenAIModels.DefaultEmbeddingsModel;
+
+    /// <summary>
+    /// Gets or sets the maximum number of characters to chunk the input into.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// At the time of writing, the maximum input tokens allowed for second-generation input embedding models
+    /// like <c>text-embedding-ada-002</c> is 8191. 1 token is ~4 chars in English, which translates to roughly 32K 
```

Comment:
This comment needs to be udpated, yes?

Reply:
Looking at the embedding model deployments availability, should n't we keep text-embedding-ada-002 as default and use those token limits of the same?   - https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#standard-deployment-model-availability

Reply:
I see. I didn't realize that Azure OpenAI was so far behind public OpenAI wrt embeddings. In this case, I agree that defaulting to ada-002 makes sense.